src/components/ButtonPanel.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-unused-vars */
+import Button from './Button';
+
+/* eslint-enable no-unused-vars */
+function ButtonPanel() {
+  return (
+    <div class = "calc">
+      <div className = "btn-show">
+      < Button name = 'AC'/>
+      < Button name = '+/-'/>
+      < Button name = '%'/>
+      < Button name = '÷'/>
+      </div>
+
+    <div className = "btn-show">
+      < Button name = '7'/>
+      < Button name = '8'/>
+      < Button name = '9'/>
+      < Button name = 'X'/>
+    </div>
+    <div className = "btn-show">
+      < Button name = '4'/>
+      < Button name = '5'/>
+      < Button name = '6'/>
+      < Button name = '-'/>
+    </div>
+    <div className = "btn-show">
+      < Button name = '1'/>
+      < Button name = '2'/>
+      < Button name = '3'/>
+      < Button name = '+'/>
+    </div>
+    <div className = "btn-show">
+      < Button name = '0'/>
+      < Button name = '.'/>
```

Comment:
- [ ] Rather than all these please group buttons in an array and map through it:
```
Group 1: AC, +/-, %, ÷
Group 2: 7, 8, 9, X
Group 3: 4, 5, 6, -
Group 4: 1, 2, 3, +
Group 5: 0, ., =
```